samza/indexer/src/main/java/org/sunbird/jobs/samza/service/Service.java

```diff
@@ -0,0 +1,111 @@
+package org.sunbird.jobs.samza.service;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Map.Entry;
+import java.util.Set;
+
+import org.sunbird.common.ElasticSearchUtil;
+import org.sunbird.common.models.util.LoggerEnum;
+import org.sunbird.common.models.util.ProjectLogger;
+import org.sunbird.common.models.util.ProjectUtil;
+import org.sunbird.models.Constants;
+import org.sunbird.models.Message;
+import org.sunbird.models.MessageCreator;
+
+import com.typesafe.config.ConfigFactory;
+import com.typesafe.config.ConfigValue;
+
+/**
+ * @author iostream04
+ *
+ */
+public class Service {
+  private static final String INDEX = ProjectUtil.EsIndex.sunbird.getIndexName();
+ 
+  private static MessageCreator messageCreator = null;
+  private final static String confFile = "indexMapping.conf";
+  private static com.typesafe.config.Config config = null;
+  private static Map<String, String> properties = null;
+
+  public Service() {
```

Comment:
Define following configuration in lms-mw.

eventSync = {
default: false,
location: true
}

Use above config in respective micro-service to change from actor sync to event sync.